### ISSUE TITLE
Support for disabling ssl certificate verification

### ISSUE THREAD

**Issue by assaftibm** [IBM/ibm-generative-ai#113]
**Context**

I want to disable the SSL certificate verification.

This is needed for accessing models that are hosted behind a proxy. For example, the Llama 2 models, as described by Chris Milite  (milite@us.ibm.com) on [Slack](https://ibm-research.slack.com/archives/C05JD18QVPA/p1690220795076509?thread_ts=1690211488.484009&cid=C05JD18QVPA). So the user has two options:
1. To manually add certificates on his/her machine.
2. To disable the certificate verification. Ian Molloy (molloyim@us.ibm.com) showed how to do that in gen-ai:

![image](https://github.com/IBM/ibm-generative-ai/assets/28494432/dd27f66d-01c9-4430-84cf-90c9180a3335)

This Feature request is about option 2 - to add support for disabling the certificate verification.

**Alternatives**

No

**Has the feature been requested before?**

No

**If the feature request is approved, would you be willing to submit a PR?**

No

**Comment by Tomas2D** [comment]
Hey @assaftibm, this functionality will be released soon (on Monday, probably).

You can then modify all options passed to the `httpx` library.

```python
from genai.utils.http_provider import HttpProvider

HttpProvider.default_http_client_options["verify"] = False
HttpProvider.default_http_transport_options["verify"] = False

# YOUR CODE GOES HERE
```

**Comment by Tomas2D** [comment]
Released in [v0.2.6](https://github.com/IBM/ibm-generative-ai/releases/tag/v0.2.6) 🚀
